api/pkg/apis/v1alpha1/managers/campaigns/campaigns-manager.go

```diff
@@ -171,3 +220,33 @@ func (t *CampaignsManager) ListState(ctx context.Context, namespace string) ([]m
 	}
 	return ret, nil
 }
+
+func (t *CampaignsManager) GetLatestState(ctx context.Context, id string, namespace string) (model.CampaignState, error) {
```

Comment:
Campagins manager doesn't know about versioning. Versioning should be handled by CampaignContainer, which is designed to handle versioning.